index.js

```diff
@@ -57,11 +66,31 @@ function tileSolveRequest(reqObj, socket){
 function noMoreMovesRequest(user, socket){
 	if(answers.length === 0){
 		console.log('No more moves found by user: ' + user);
-		io.emit('setupPhase');
+		if(score[reqUser.username]){
+			score[reqUser.username].points += 2;
+		} else {
+			score[reqUser.username] = {
+				user: reqUser,
+				points: 2
+			};
+		}
+		phase = 'setupPhase';
+		io.emit(phase, score);
 		setTimeout(setupPhase, 5000);
 	} else {
 		console.log('There are still more moves: ' + user);
 		socket.emit('errorNoMoreMovesRequest');
+		if(score[reqUser.username]){
+			score[reqUser.username].points -= 1;
+			if(score[reqUser.username].points < 0){
+				score[reqUser.username].points = 0;
+			}
```

Comment:
why not do `points -= 1` after a `points > 0` check and then use `else`? Saves one write operation